### DISCUSSION TITLE
A98FD77F-9820-1D18-B530-1A693D95AF2C

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#164] [General]
# A98FD77F-9820-1D18-B530-1A693D95AF2C

This addon enhances Saturn's moon Rhea, including:

- xyzv trajectory from 2004 to 2018 to cover the Cassini mission, ensuring more accurate flybys.
- 3D model based on Cassini data.
- Alternative texture of enhanced colors.
- Also includes the (HYPOTHETICAL!) Rhea ring system proposed in 2008. Note that the rings are very, very thin. So, in Celestia, you'll need to get very close to Rhea's equator in order to see them.

https://celestia.mobi/addon?item=A98FD77F-9820-1D18-B530-1A693D95AF2C

**Comment by SevenSpheres** [comment]
While there is speculation that Rhea and/or Iapetus may have had rings in the past, there is no evidence that Rhea presently has rings. See Tiscareno et al. (2010), ["Cassini imaging search rules out rings around Rhea"](https://ui.adsabs.harvard.edu/abs/2010GeoRL..3714205T/abstract).
